lib/eChronicler_web/templates/page/index.html.heex

```diff
@@ -5,37 +5,20 @@
 
 <section class="row">
   <article class="column">
-    <h2>Resources</h2>
+    <h2>Post Entry</h2>
```

Comment:
Given my route change request above, this template can probably end up being deleted.

Reply:
Done!